new_tab/about.html

```diff
@@ -0,0 +1,159 @@
+            <li class="subscribe">Subscribe</li>
+          </ul>
+        </div>
+      </nav>
+
+      <nav class="header1">
+        <ul>
+          <li><ion-icon name="logo-facebook"></ion-icon></li>
+          <li><ion-icon name="logo-twitter"></ion-icon></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </nav>
+      <nav class="header2">
+        <div class="navbarDesktop">
+          <span
+            >GLCon23'<ion-icon id="naveicon" name="aperture-outline"></ion-icon
+          ></span>
+          <ul>
+            <li id="about"><a href="../index.html"> Homepage</a></li>
+            <li><a href="#customBackground">About</a></li>
+            <li><a href="#Location">Location</a></li>
+            <li><a href="#sponsors">Sponsor</a></li>
+          </ul>
+
+          <button class="navbutt">Campaign</button>
+        </div>
+      </nav>
+
+      <nav>
+        <div class="slidingMenu">
+          <div class="closeMenu">
+            <img src="../Images/WindowDisabled.png" alt="WindowDisabled" />
+          </div>
+          <ul>
+            <li><a href="../index.html">Home</a></li>
+            <li><a href="#"> About</a></li>
+            <li><a href="../index.html#programContainer">Program</a></li>
+            <li><a href="../index.html#featuredSpeakers">Speakers</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+      <div id="customBackground" class="customBackground">
+        <div class="aboutPage">
+          <h2>About Us</h2>
+        </div>
+        <p class="aboutPara">
+          The Global Leaders' Conference brings together a community of expert,
+          intellectuals and scholars who make up the Global leaders' fratenity
+          affiliate network in a diffrent country every two years. This year we
+          hope to expand our scope across the entire Microverse reaching out to
+          the globe. Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          Perferendis consequatur, ratione quod omnis ipsa aperiam quaerat
+          facilis recusandae quas hic saepe dolore a debitis, quos inventore
+          commodi nihil reiciendis consectetur!
+        </p>
+
+        <div class="contactSec">
+          <p>
+            Please contact us per Email for any further questions. We would
+            really be glad to assist you! <br />
+            <u> <b>globaleadercon@yahoo.com.</b> </u> <br />
+            or please call +1354 26569 554
+          </p>
+        </div>
+
+        <div class="News">
+          <h3>
+            News Updates <br />
+            Global Leaders' Conference
+          </h3>
+          <p>
+            GLCon23' <ion-icon id="naveicon" name="aperture-outline"></ion-icon>
+          </p>
+        </div>
+
+        <h4 id="Location" ><u>See Past and Future Events! </u></h4>
+        <span>
+          Take a quick glance @ the latest Conference held in Ghana, Accra
+        </span>
+

```

Comment:
## Desktop Version

-  [ ] As I mentioned above :point_up_2: please stick to the Behance design but there are some spacing and alignments issues and also font size and font family are not the same as the Behance design so kindly fix all these issues :+1: check below :point_down:  the comparison between your design and Behance design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-07-07 15-59-03](https://github.com/Mike47ip/JavascriptCapstone/assets/45226848/7f1b1c0d-7ad5-4fee-ae40-995a51e6a569)

**Behance Design**
![Screenshot from 2023-07-07 15-58-52](https://github.com/Mike47ip/JavascriptCapstone/assets/45226848/84ce9bb6-b0cc-4749-9d01-2ca3cf95b524)

## Mobile Version

-  [ ] As I mentioned above :point_up_2: please stick to the Behance design but there are some spacing and alignments issues and also font size and font family are not the same as the Behance design so kindly fix all these issues :+1: check below :point_down:  the comparison between your design and Behance design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-07-07 16-03-08](https://github.com/Mike47ip/JavascriptCapstone/assets/45226848/4e3f0b42-a922-4126-acd1-5cfd066eb4df)

**Behance Design**
![Screenshot from 2023-07-07 16-03-22](https://github.com/Mike47ip/JavascriptCapstone/assets/45226848/2cf58bb6-9d50-42e0-ab91-c8c181145c70)